### DISCUSSION TITLE
Updated program - forcing Apple Silicon onto Intel mini

### DISCUSSION THREAD

**Discussion by rleaver152** [theophanemayaud/video-simili-duplicate-cleaner#127] [General]
The latest update seems to force an Apple Silicon compilation onto my Mac Mini 2014 intel (Monterey) so I have to kludge a bash solution to replace it with a backup copy.    Not sure why it is doing this.

**Comment by theophanemayaud** [comment]
Hey ! I’m not sure what you mean by « force an Apple Silicon compilation », could you explain a bit more please ? :)

**Reply by rleaver152** [reply]
Sure - the automatic Apple Store software update is pushing only the Apple Silicon version to my Intel machine which then says it cannot run on it.   I dont know why.  Is the update only Apple Silicon?  then its seeing it as an update but pushing to the wrong platform?

**Reply by theophanemayaud** [reply]
What does it say when you try to run the app ? It should run fine regardless of Apple silicon, but maybe there is a problem with the macOS version compatibility. I don’t have a Mac running an older macOS version so I can’t check easily. I’ll check what I can do, but it’ll probably be in the like of disabling updates for older macOS versions… 😰

**Reply by rleaver152** [reply]
Exactly a pop up screen "You can’t use this version of the application “Video simili duplicate cleaner.app” with this version of macOS."

**Reply by rleaver152** [reply]
"You have macOS 12.7.2. The application requires macOS 13.0 or later."

**Reply by rleaver152** [reply]
(apologies - the Apple Silcon thing got into my head from the update comments)   My old Mac Mini 2014 wont update to 13.0

**Reply by theophanemayaud** [reply]
Oh no, I feared because I now built to be also compatible natively with Apple silicon that there would be retrocompatibility problems with older macOS versions (although Monterey is not old…). I’ll do a quick fix of removing appp store support for the older versions so it doesn’t update in your case. 
longer term I’ll check if there’s a way to stay functioning on the last few macOS versions. 

Thanks for reporting this !!!

**Comment by theophanemayaud** [comment]
So I worked on this issue this weekend and the only solution I found for now is to build back to only intel architecture. I published a new version to the App Store, as soon as it's accepted by apple I hope it'll solve your problems like also #126, for which I tested the specific file that caused the crash and now doesn't crash.

**Comment by rleaver152** [comment]
Many thanks - much appreciated.      If I may,   can you suggest best settings for a no false positives solution.   I know its not a perfect science,  but  the most stringent even if slow settings would be greatly appreciated.   I run on defaults usually but find it is picking up quite a few similar but not identical home videos

thanks

Richard

**Reply by theophanemayaud** [reply]
There's no absolute rule, you have to see what works best for you. I use different settings on each batch of video I process until I feel confident and turn on auto. Feel free to report here if you find a good setting !
